cryptonote_utils/cryptonote_utils.js

```diff
@@ -1688,7 +1688,7 @@ var cnUtil = function(currencyConfig)
 			}
 			
 			// send change to ourselves
-			else if(dsts[i].keys.view == keys.view.pub) {
+			if(dsts[i].keys.view == keys.view.pub) {
```

Comment:
ok, makes more sense now (without vouching for correctness)... one minor nitpick would be to squash these amendment commits with the original

Reply:
I just realized though I can squash as I merge. My mistake.